test/ini_files/cg_any_alpha.ini

```diff
@@ -15,6 +15,7 @@ tibanna_cwls_bucket = ${TIBANNA_CWLS_BUCKET}
 tibanna_output_bucket = ${TIBANNA_OUTPUT_BUCKET}
 app_kind = ${APP_KIND}
 app_deployment = ${APP_DEPLOYMENT}
+s3_encrypt_key_id = ${S3_ENCRYPT_KEY_ID}
```

Comment:
Note that this must also be done in cgap-portal.

Reply:
Right. This is just testing that it will work to do that.